common/common.scss

```diff
@@ -665,7 +669,7 @@ td.poster-names a {
     padding: 6px 12px;
     font-size: $font-up-1;
     transition: none;
-    height: $--global-nav-height;
+    height: 42px;
```

Comment:
We really should not be hard-coding, unless put in a global variable.

Reply:
I understand,

In the current context, I was wary of changing the global variable itself not knowing exactly what it was going to affect but thinking it might have impacts on many pages while not knowing myself how to compare the before/after versions of it aside from visually trying to assess it...

I just gave it a shot at setting the variable to 42px instead of 38px and it seems to be working fine but the variable is used 7 times including in padding in media requests which I am worried could cause regressions in other places without us realizing...

What do you believe should be done?

Reply:
If the height is removed it still causes issues as the button expands substantially...  Think I should try to figure it out removing it completely and substitute the effect by adding margins to it?

Reply:
You are correct to not immediately change the global variable, if it is used in other places, without knowing how it will affect the page. However, the reason for having a global variable is to keep all related elements matched.

My advice is still what I mentioned in my last comment on the issue, though.

Reply:
My mistake for not being detailed enough.  Here are the results of my playing around with this.

**Give the ol a margin: 0 0 var(--nav-space)**

    The ol already has a margin of 0.  I'm assuming you mean the Div within the li within it?
    
	    If so, the margin comes from default theme.  The only way I can think of doing this without editing discourse's 
 structure is to override it with :
		    
		    div.select-kit-header {
		    margin-bottom: 0 !important;
		    }
		    
	    In any case, this margin does seem to impact the height of the last 3 buttons with a minimal difference being obtained setting it to 0 but still not bridging the gap between sizes of the first and last 3 buttons.  This is why I found it being resolved by combining this one with an increase in height of the .list-controls .combo-box .combo-box-header from 38px to 42px.

	TLDR: It doesn't appear to make it possible to fix the gap in sizes on it's own.

**Get rid of the .category-breadcrumb.select-kit-header margin-bottom: var(--nav-space)**

	This is what is discussed above unless i am missing another margin somewhere else

**Get rid of the .list-controls .combo-box .combo-box-header height: 38px**

	Getting rid of this makes the button notably larger as well as Potentially impacting many other things or requiring a !important?

Tell me if I am missing something.  I understand the need to avoid hardcoding in which case something like  

height: $--global-nav-height + 4px; or even height: $--global-nav-height + $--global-nav-height/9.5;

Could do the trick instead of keeping proportions and reliance on global height?

Reply:
I will need to look into it with an instance of Discourse, in order to better be able to help.

Reply:
Maybe we could use the one I setup locally via a quick zoom or discord session using screen share or something similar?  Let me know of a time if that works for you.  I'd like to get a chance to participate and get to have an outside perspective on it!

Reply:
See this: https://github.com/freeCodeCamp/freeCodeCamp/issues/41559#issuecomment-812508219